### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)
w

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

File: scripts/change_url [hunk 1]
```diff
@@ -44,7 +44,7 @@ ynh_store_file_checksum --file="/etc/transmission-daemon/settings.json"
 #=================================================
 ynh_script_progression --message="Starting a systemd service..." --weight=2
 
-ynh_systemd_action --service_name=transmission-daemon --action="start"
+ynh_systemd_action --service_name=transmission-daemon --action="start" --log_path="systemd"
```
Comment (Salamandar) [inline scripts/change_url:47]: install script says /var/log/$app/$app.log ?

**Review by Salamandar — COMMENTED** [review]
--log_path is inconsistent, is it wanted ?

Apart from that, nice.

File: scripts/change_url (hunk 1 above)
Comment (ericgaspar) [reply scripts/change_url:47]: `--log_path` seems consistent 🤔
https://github.com/YunoHost-Apps/transmission_ynh/blob/5c8a74721bc07751004ccaa7dcccfe511a5ed824/scripts/install#L108

**Pull request merged**